### PR TITLE
Added Transaction structs

### PR DESCRIPTION
**Pull request by jared-fraser**
Its possible to pull transaction details with an order so I've added the structs.

### CODE REVIEW

File: order_test.go [hunk 1]
```diff
@@ -102,6 +119,31 @@ func TestOrderGet(t *testing.T) {
...
+	}
+
+	orderTests(t, *order)
+
+	// Check transactions is not nil
+	if order.Transactions == nil {
+		t.Error("Expected Transactions to not be nil")
+	}
+	if len(order.Transactions) != 1 {
+		t.Errorf("Expected Transactions to have 1 transaction but recieved %v", len(order.Transactions))
```
Comment (dlebech) [inline order_test.go]: Small typo: `received`

**Review by dlebech — COMMENTED** [review]
Thanks for the PR!

I'm surprised you were able to get transactions as part of the orders/X.json call. It's not documented [in the orders API](https://help.shopify.com/api/reference/order), and on my test store, no transactions are returned from a call to `/admin/orders/1234.json`. I have to do `/admin/orders/1234/transactions.json` to get the transactions. Could you clarify how you produced the fixture that you included in this PR? I want to make sure we avoid confusion.

**Comment by jared-fraser** [comment]
Fixed spelling mistake and updated test to show the struct to pass through to bring back transactions for an order.

It is an undocumented API feature which means you can't be sure it'll always be supported but its too handy to pass up when syncing orders and transaction with the api limits.

**Review by dlebech — APPROVED** [review]
Thanks for clarifying @jared-fraser :+1:

**Comment by jared-fraser** [comment]
thanks, @dlebech. do you have any more issues that you could label for hacktoberfest (https://hacktoberfest.digitalocean.com/)? I'll do some more contributions.

**Comment by dlebech** [comment]
@jared-fraser haha, what a wonderful idea. I never heard of Hacktoberfest before :) Do you need the issue to be tagged beforehand or does a tagged PR count as well? Either way, if you're up for it, feel free to add some issues for missing resources and I can label them as Hacktoberfest for you. I've added the label to one existing issue #22. Other resources that would probably be nice to have are an enhanced customer model, a `Transactions()` function on the orders struct (for those times where you don't want to rely on the undocumented API) and a way to also fetch transactions by themselves. Any contributions to new resources are valuable. I have just been adding the ones I needed on an ad-hoc basis.

**Comment by jared-fraser** [comment]
They don't really need the label, the PR counts towards the goal. However, it brings light to the project for other people searching for that label tag and feeling like they want to contribute.

**Pull request merged**